docs/source/athena-user-guide.rst

```diff
@@ -0,0 +1,59 @@
+Athena User Guide
+=================
+
+Concepts
+--------
+
+* `SQL <https://www.w3schools.com/sql/sql_intro.asp>`_
+* `Athena Partitions <http://docs.aws.amazon.com/athena/latest/ug/partitions.html>`_
+
+
+Querying Data
+-------------
+
+After completion of the `Athena Setup <athena-setup.html>`_ and `Athena Deploy <athena-deploy.html>`_, alert data is now searchable in the Athena UI.
```

Comment:
`After .... , alerts generated from StreamAlert are now searchable in the Athena UI.`